### PR TITLE
double dash (setup & troubleshooting)

### PR DESCRIPTION
**Pull request by mateuszsrebrny**

### CODE REVIEW

File: src/pages/docs/creators/ray/troubleshooting.md [hunk 1]
```diff
@@ -155,24 +155,43 @@ Check out the [cluster yaml reference](/docs/creators/ray/cluster-yaml-reference
...
+Ray On Golem webserver
+  Not starting webserver, as it's already running
+Fetched IP: 192.168.0.3
+root@192.168.0.3's password: 
+``` 
 
 {% solution %}
 
-Solution
+`ray submit` has its own arguments. If your script arguments are named the same, Ray will get confused.
```
Comment (shadeofblue) [inline src/pages/docs/creators/ray/troubleshooting.md:183]:
```suggestion
`ray submit` and other ray commands have its own arguments. If your script arguments are named the same, Ray will get confused.
```

File: src/pages/docs/creators/ray/troubleshooting.md [hunk 2]
```diff
@@ -155,24 +155,43 @@ Check out the [cluster yaml reference](/docs/creators/ray/cluster-yaml-reference
...
+
+In this example `-n` stands for cluster name, which we don't support yet - hence random asking for SSH password.
```
Comment (shadeofblue) [inline src/pages/docs/creators/ray/troubleshooting.md:185]:
```suggestion
In this example `-n` stands for cluster name, which we don't support yet - hence the unexpected request for the SSH password.
```

File: src/pages/docs/creators/ray/troubleshooting.md [hunk 3]
```diff
@@ -155,24 +155,43 @@ Check out the [cluster yaml reference](/docs/creators/ray/cluster-yaml-reference
...
+
+The solution is to precede your arguments with `--`:
+```bash
+ray submit golem-cluster.yaml examples/simple-task.py -- -n 20
+```
```
Comment (shadeofblue) [inline src/pages/docs/creators/ray/troubleshooting.md:191]:
```suggestion
This informs Ray that everything after the double dash is not to be interpreted, but instead passed as-is to the executed script.
```

**Review by shadeofblue — COMMENTED** [review]
lgtm, added a few comments :)

**Comment by mateuszsrebrny** [comment]
@cryptobench a small fix for non urgent merging :)

**Comment by approxit** [comment]
I'm missing the `--` info in https://docs.golem.network/docs/creators/ray/conversion-to-ray-on-golem-tutorial

**Comment by mateuszsrebrny** [comment]
> I'm missing the `--` info in https://docs.golem.network/docs/creators/ray/conversion-to-ray-on-golem-tutorial

How and where would you put it @approxit ?

I left this one out as `dds.py` doesn't require arguments and I thought mentioning it will add complexity without profit (as there is no opportunity to try it out) 🤔

**Comment by approxit** [comment]
> > I'm missing the `--` info in https://docs.golem.network/docs/creators/ray/conversion-to-ray-on-golem-tutorial
> 
> How and where would you put it @approxit ?
> 
> I left this one out as `dds.py` doesn't require arguments and I thought mentioning it will add complexity without profit (as there is no opportunity to try it out) 🤔

Ideally on example with arguments... But to avoid rewriting nearly whole article, notice after first `ray submit` would do the job.

**Comment by mateuszsrebrny** [comment]
ok. @cryptobench now we are really ready to merge this :dancers:

**Review by cryptobench — APPROVED** [review]

**Pull request merged**